.platform.app.yaml

```diff
@@ -29,6 +29,9 @@ hooks:
     ruby -v
     curl -sS https://platform.sh/cli/installer | php
     gem install bundler
+
+    sed -i "s/host:.*/host: https:\/\/www---pr-263-pugnbma-ovkfaqrdltr5s.us.platform.sh/" app/config/sculpin_site.yml
```

Comment:
this line is temporary and it is only for testing this PR. should be removed before merging